lib/tilelivecopy.js

```diff
@@ -45,9 +47,11 @@ function tilelivecopy(srcUri, s3url, options, callback) {
         options.close = true;
         if (scheme === 'list') options.listStream = src.createZXYStream();
 
-        if (options.stats) {
-          options.transform = stats;
-        }
+        var transforms = [];
+
+        if (options.stats) transforms.push(stats);
+        if (options.type === 'mbtiles') transforms.push(MigrationStream());
+        if (transforms.length > 0) options.transform = combiner(transforms);
```

Comment:
This looks fancy, mind explaining what's going on here @jakepruitt? From the sounds of `combiner` I imagine this is how we are sending two streams through `tilelive.copy`?